src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -346,8 +346,18 @@ private void WriteHeader()
 
         private void WriteHelp()
         {
-            //TBD
-            Console.WriteLine("In Development");
+            Console.Out.WriteLine(SR.Format(SR.HelpDescription));
+            Console.Out.WriteLine(SR.Format(SR.HelpUsage, this.GetType().Assembly.GetName().Name));
+            Console.Out.WriteLine(SR.Format(SR.HelpDevOptions));
+            Console.Out.WriteLine(SR.Format(SR.HelpAssembly, "/assembly:", "/a:"));
+            Console.Out.WriteLine(SR.Format(SR.HelpType, "/type:", "/t:"));
```

Comment:
Are all these options supported by upcoming release? If not, we should not include options that we don't support yet.

Reply:
Will remove debug. All others are supported.